kernel/fs/fs_utils.c

```diff
@@ -109,14 +114,14 @@ ssize_t fs_load(const char * src, void ** out_ptr) {
     }
 
     /* Did we get it all? If not, realloc the buffer */
-    if(left > 0) {
-        *out_ptr = realloc(data, total);
-
-        if(*out_ptr == NULL)
-            *out_ptr = data;
+    if (left > 0) {
+        void * new_data = realloc(data, total);
+        if (new_data != NULL) {
+            data = new_data;
+        }
```

Comment:
In addition to the below stylistic fixes, please move the declaration of `new_data` to the beginning of the function.

```suggestion
    if(left > 0) {
        void * new_data = realloc(data, total);
        if(new_data != NULL) {
            data = new_data;
        }
```